### PR TITLE
feat(FWS-57): Ability to view coach notes by user

### PR DESCRIPTION
**Pull request by emilegirardGit**
JIRA: link to jira ticket
https://champlainsaintlambert.atlassian.net/browse/FWS-57

What is the ticket about and why are we making this change?

This ticket was about adding the Coach Notes in the member profile. This was added through a coach note tab on the profile sidebar.

What are the various changes and what other modules do those changes affect?

-Fix styling for member profile
-Fix tab in member profile
-Added coach notes tab
-Display list of coach notes

UI Change:

image

### CODE REVIEW

**Review by Eli293 — COMMENTED** [review]
Should we be using a different sub domain for coach notes?

File: src/test/java/com/fitwsarah/fitwsarah/appointmentsubdomain/presentationlayer/AppointmentControllerIntegrationTest.java [hunk 1]
```diff
@@ -69,14 +71,6 @@ void getAppointmentByAppointmentId_Should_Return_Unauthorized() throws Exception
                 .andExpect(status().isUnauthorized());
     }
 
-    @Test
-    public void getAllAppointmentsByAccountId_ShouldReturnIsOK() throws Exception {
```
Comment (Eli293) [inline src/test/java/com/fitwsarah/fitwsarah/appointmentsubdomain/presentationlayer/AppointmentControllerIntegrationTest.java:73]: Is this ok to delete?

**Review by Eli293 — APPROVED** [review]
Nice job.

File: fitwsarah_frontend/src/components/CoachNote/CoachNote.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    };
+
+
+    return (
+        <div className="notes-container">
+            <h1>Coach Notes</h1>
+            <div className="note-row">
+                {contents && contents.map((content, index) => (
+                    <div className="note-card" key={index}>
+                    <div className="note-title">Coach Note {index + 1}</div>
```
Comment (AlexeiTimbro) [inline fitwsarah_frontend/src/components/CoachNote/CoachNote.js:58]: On this line why did you make this line using the index +1 "Coach Note {index + 1}"? And what does it do? I'm curious.

**Pull request merged**